app/assets/javascripts/discourse/app/components/bookmark-menu.gjs

```diff
@@ -2,7 +2,6 @@ import Component from "@glimmer/component";
 import { tracked } from "@glimmer/tracking";
```

Comment:
Not sure about this one, it did appear in some retained trace and that looked suspicious, given I think the new code is just better, I didn't spend much more time thinking about it